pilot/pkg/bootstrap/server.go

```diff
@@ -915,6 +917,15 @@ func (s *Server) getRootCertificate(tlsOptions TLSOptions) (*x509.CertPool, erro
 	}
 	cp := x509.NewCertPool()
 	cp.AppendCertsFromPEM(rootCertBytes)
+
+	if features.AuthnSpiffeBundlePath != "" {
```

Comment:
Not sure what is the use case - but let's avoid repeating past mistakes. 

In most cases we need more than one certificate - either make the 'spiffee bundle path' a list, so we can fetch multiple docs, or parse multiple certs from the bundle ( it seems to be only one ). 

Also not entirely sure why this is in pilot - and not pilot-agent ( or at least usable by both ). Is it just so Istiod combines all the roots ( from Secret, config - and Spiffee URLs ) - and distributes them as the config map ?

Reply:
Sounds reasonable. I'll update the PR tomorrow with ability to load root certs from multiple endpoints.